File: slo.go
//usr/bin/env go run $0 $@; exit $?
package main

import "fmt"

type Point struct {
	x int
	y int
}

func main() {
	total := 0
	for i := 0; i < 1000000; i++ {
		p := Point{2, 3}
		total = total + p.x + p.y
	}
	fmt.Println(total)
}

File: slo.java
//usr/bin/env javac $0; cd $(dirname $0); java slo; exit $?
class Point {
  public int x, y;
  public Point(int x, int y) {
    this.x = x;
    this.y = y;
  }
}

public class slo {
  public static void main(String[] args) {
    int total = 0;
    for (int i = 0; i < 1000000; i++) {
      Point p = new Point(2, 3);
      total = total + p.x + p.y;
    }
    System.out.println(total);
  }
}
